.gitignore

```diff
@@ -402,4 +402,6 @@ FodyWeavers.xsd
 ### VisualStudio Patch ###
 # Additional files built by Visual Studio
 
-# End of https://www.toptal.com/developers/gitignore/api/visualstudio
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/visualstudio
+
+appsettings.json
```

Comment:
we can consider using appsettings.Development.json or other local version in next versions, and allow the default appsettings.json